src/scripts/ui/WarehouseBuildingBody.tsx

```diff
@@ -62,6 +64,26 @@ export function WarehouseBuildingBody({ gameState, xy }: IBuildingComponentProps
                      )}
                   </div>
                </div>
+               <div className="sep10" />
+               <div className="separator"></div>
+               <div className="row">
+                  <div className="f1">{t(L.AutopilotStoragePercentageDesc)}</div>
+                  <div className="text-strong">{formatPercent(options.autopilotPercentage)}</div>
+               </div>
+               <input
+                  type="range"
+                  min="0"
```

Comment:
I am not sure about allowing 0-100%. Technically there's no harm but perhaps it's better to limit to like 90%-100% to avoid unintended effects?